scripts/train.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Clear notebook before saving / commit. There is also a special hook `jupyter nbconvert --clear-output --inplace scripts/*.ipynb`